website/content/tutorials/1-getting-started/_index.md

```diff
@@ -74,19 +77,27 @@ We will now add to the [`assembly`](/specification/concepts/terminology/#assembl
   <json-base-uri>http://example.com/ns/computer</json-base-uri>
   <define-assembly name="computer">
     <formal-name>Computer Assembly</formal-name>
-    <description>A container object for a computer, its parts, and its sub-parts.</description>
+    <description>A container object for a computer, its parts, and its
+      sub-parts.</description>
     <root-name>computer</root-name>
     <define-flag name="id" as-type="string" required="yes">
         <formal-name>Computer Identifier</formal-name>
-        <description>An identifier for classifying a unique make and model of computer.</description>
+        <description>An identifier for classifying a unique make and model of
+          computer.</description>
     </define-flag>
   </define-assembly>
 </METASCHEMA>
 ```
 
-With the definition of the first `assembly` our Metaschema definition takes shape. Firstly, we now add the `root-name` element to identify for Metaschema-enabled tools this `assembly`, whether there are one or more top-level ones defined, is the root element of our data models. This will come in handy later. In the `assembly` declaration, we have defined a `computer` that must have an `id`, the value of which must be `string`. With Metaschema-enabled tooling, we can have information about a computer like below in their respective JSON, XML, and YAML data formats.
+With the definition of the first `<define-assembly>`, representing a simple computer information element, our Metaschema module starts to take shape.
+
+In the `<define-assembly>` declaration we have defined an object named `computer` (see line 8).
 
-An instance for the model above is below in JSON, XML, and YAML data formats.
+The computer object must have an `id`, the value of which must be [`string`](/specification/datatypes/#string) (see lines 12-15).
+
+The `<root-name>` element is used to identify to Metaschema-enabled tools that this assembly is allowed to be the root element of our data model (see line 11). This will come in handy later.
```

Comment:
Minor, non-blocking: it is allowed to be or must be?

```suggestion
The `<root-name>` element is used to identify to Metaschema-enabled tools that this assembly must be the root element of our data model (see line 11). This will come in handy later.
```

Reply:
```suggestion
The `<root-name>` element is used to identify to Metaschema-enabled tools that this assembly MUST be the root element of our data model (see line 11). This will come in handy later.
```